shell/app-shell/app-shell.js

```diff
@@ -91,8 +91,7 @@ const template = html`
     arc="{{arc}}"
     metadata="{{metadata}}"
     share="{{share}}"
-    plans="{{plans}}"
-    step="{{step}}"
+    planificator="{{planificator}}"
```

Comment:
It's important to keep components as ignorant as possible, which simplifies contracts and makes things wildly easier to maintain and remix. In particular we want to avoid passing API objects around in the component tree, and avoid listening to signals (callbacks) in more than one place. 

In other words, ideally only one object knows about Planificator and mediates command and control amongst the other components that deal only in plain data. 

I'll work on a more concrete suggestion, but I wanted to give you a heads up that we'll probably need to do some rearranging on how the new goodies are plumbed into app-shell.

Reply:
Scott, thanks for your feedback. I think I understand what you mean, no need for a concrete suggestion, let me rework the change first.

Reply:
(I'm still working on some additional planificator tests, but) the shell hooks are updated and a lot less disruptive now. PTAL and tell me what you think.